core/src/main/java/org/apache/accumulo/core/client/impl/ScannerIterator.java

```diff
@@ -92,13 +92,13 @@ public void run() {
           return;
         }
       } catch (IsolationException | ScanTimedOutException | AccumuloException | AccumuloSecurityException | TableDeletedException | TableOfflineException e) {
-        log.trace(e.getMessage(), e);
+        log.trace("{}", e.getMessage(), e);
```

Comment:
General question: is it worth inserting the formatting string when there isn't any concatenation happening?